package.json

```diff
@@ -8,12 +8,40 @@
     "build:app": "vuepress build docs",
     "build:lambda": "netlify-lambda build lambda",
     "docs:build": "vuepress build docs",
-    "postinstall": "netlify-lambda install"
+    "postinstall": "netlify-lambda install",
+    "test": "./node_modules/.bin/jest --silent"
   },
   "author": "Alan Woo (Bin Studio) <alan@bin.am>",
   "license": "MIT",
+  "jest": {
+    "moduleFileExtensions": [
+      "js",
+      "json",
+      "vue"
+    ],
+    "transform": {
+      ".*\\.(vue)$": "vue-jest",
+      "^.+\\.js$": "<rootDir>/node_modules/babel-jest"
+    },
+    "moduleNameMapper": {
+      "^@/(.*)$": "<rootDir>/docs/.vuepress/$1",
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__test__/mocks/FileMock.js"
+    },
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "docs/.vuepress/theme/**/*.{js,vue}",
+      "!**/node_modules/**"
+    ]
+  },
   "devDependencies": {
     "@vuepress/plugin-active-header-links": "^1.0.0-rc.1",
+    "@vue/test-utils": "^1.0.0-beta.29",
+    "babel-jest": "^24.9.0",
+    "babel-preset-env": "^1.7.0",
```

Comment:
Any reason to use `babel-preset-env` vs `@babel/preset-env`?

Reply:
nop, i'll change `babel-preset-env` in favor of `@babel/preset-env` just now!